packages/web/src/js/FileUploader.ts

```diff
@@ -265,10 +265,8 @@ class FileUploader extends BaseComponent {
     const itemButton = snippet.querySelector(`[${TEMPLATE_ELEMENT_SLOT_NAME}="button"]`);
     const hasImagePreview = Boolean((item as HTMLElement)?.dataset?.spiritImagepreview);
     const AttachmentSvgIcon = item?.querySelector('svg');
-    const AttachmentPreviewImage = snippet.querySelector('.FileUploaderAttachment__image');
+    const AttachmentPreviewImage = item?.querySelector('[data-spirit-element="attachment-image"]');
```

Comment:
You are repeating a `[data-spirit-element="attachment-image"]` selector, so you can make a constant from this value and use it on both places.

Reply:
i'll do that, thank you